lib/fluent/plugin/in_tail.rb

```diff
@@ -357,6 +359,11 @@ def convert_line_to_event(line, es, tail_watcher)
             record[@path_key] ||= tail_watcher.path unless @path_key.nil?
             es.add(time, record)
           else
+            if @enable_catch_all
+              record = {'parse_fail' => line}
+              record[@path_key] ||= tail_watcher.path unless @path_key.nil?
+              es.add(Time.now.to_i, record)
```

Comment:
We're using the Redhat version of td-agent which comes with Fluentd v0.12. Do you know when the 0.14 branch will be stable? If it's still quite long away, should I merge my branch into the 0.12 branch instead?